model/role.go

```diff
@@ -97,6 +100,59 @@ func init() {
 			PERMISSION_EDIT_BRAND,
 		},
 	}
+
+	SystemUserManagerDefaultPermissions = []string{
```

Comment:
Just making sure I understand, this is basically just refactoring these permission lists so the code is more readable?

Reply:
(And also makes it easier to add the common ancillary permissions)

Reply:
Its mostly just to expose the defaults as something that can be used by the telemetry package, before this there was no way to grab initial server defaults for a role without duplicating the stuff here